src/features/authentication/sign-up/SignUp.service.ts

```diff
@@ -14,7 +15,45 @@ export class SignUpService {
     if (await this.signUpDao.userExists(member)) {
       throw new HttpError(ResponseConflict.message, ResponseConflict.code)
     }
-
+    if (member.password) {
```

Comment:
If password is not there in the member, then you should send error message saying "Please provide password".

Without password you cannot create a member right.